### ISSUE TITLE
License key submission issue

### ISSUE THREAD

**Issue by gsacre** [Techmentis/flinkiso-lite#2]
Hi,

I have an issue after submitting my license key:
- I submit my license key
- the system finds my user identification I used to register on the website
- it asks for a password, I use the one sent to me via email (I tried also to enter a different password from the one I received by email (in case it was to create myself a new user) and got the same result)
- I get a webpage (see attachment) that asks me again for the license key

I use Ubuntu Server 15.04 base (no specific package from a specific repository).

![flinkiso installation](https://cloud.githubusercontent.com/assets/656009/8988815/c1e71e6e-36e8-11e5-9c64-e33e8820b339.png)

**Comment by Techmentis** [comment]
Hi,
I can see that there is a "Internal Error" .. can you check what's the issue in servers error log ?

**Comment by gsacre** [comment]
Thanks for your help!
It was indeed an internal web server issue:

PHP Fatal error:  Call to undefined function curl_init() in /var/www/flinkiso/app/Controller/UsersController.php on line 1621, referer: http://192.168.1.240/users/register

For reference, it was a PHP modue missing.
So I had to edit my php.ini and add the following line: "extension=php_curl.so"
Then, I had to install the module: sudo apt-get install php5-curl
I restarted apache (not sure it was necessary but you never know) and then the installation completed successfully.

Maybe add this to the installation help page that ships with the software so that others will not face it as well.

Thanks for your help!
